src/templates/content.jsx

```diff
@@ -0,0 +1,57 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Utils from '../utils/utils';
+
+const propTypes = {
+    as: PropTypes.oneOf(['div', 'header', 'main', 'section']),
+    children: PropTypes.node,
+    className: PropTypes.string,
+    id: PropTypes.string,
+    isFiltersRailOpen: PropTypes.bool,
+    moduleType: PropTypes.oneOf(['drawer', 'page']).isRequired,
+    scrollable: PropTypes.bool,
+    style: PropTypes.shape({}),
+};
+
+const defaultProps = {
+    as: 'section',
+    children: undefined,
+    className: undefined,
+    id: undefined,
+    isFiltersRailOpen: false,
+    scrollable: false,
+    style: {},
+};
+
+function PageContent(props) {
```

Comment:
Should this simply be `Content` rather than `PageContent`?

Reply:
Yes, it should. Fixing.